README.dev.md

```diff
@@ -103,6 +102,21 @@ git clone https://github.com/your-name/TulipaEnergyModel.jl                   #
 git remote add upstream https://github.com/TulipaEnergy/TulipaEnergyModel.jl  # use the original repository URL
 ```
 
+Check that your origin and upstream are correct:
+
+```bash
+git remote -v
+```
+
+You should see something similar to:
+![Screenshot of remote names, showing origin and upstream](docs/images/Remotes.PNG)
```

Comment:
```suggestion
![Screenshot of remote names, showing origin and upstream](docs/images/Remotes.png)
```
Can we rename the file to use lowercase `png`? Shouldn't make any difference, but I have a thing. Also, the other files in that folder have lowercase extension.